sbt_common/display/src/test/scala/uk/ac/wellcome/display/models/v1/DisplayWorkV1Test.scala

```diff
@@ -67,73 +68,6 @@ class DisplayWorkV1Test extends FunSpec with Matchers {
     displayWork.identifiers shouldBe Some(List())
   }
 
-  describe("publishers") {
-    it("parses a work without any publishers") {
```

Comment:
Why are these tests deleted? Isn't v1 supposed to stay as it was?

Reply:
As discussed, V1 DisplayWorks should have no publishers!